test/e2e/tests/desk/inspectDialog.spec.ts

```diff
@@ -1,37 +1,32 @@
+import {expect} from '@playwright/test'
 import {test} from '@sanity/test'
 
 const INSPECT_KEY = 'studio.structure-tool.inspect-view-mode'
 
-test('clicking inspect mode sets value in storage', async ({
-  page,
-  sanityClient,
-  createDraftDocument,
-}) => {
+test('clicking inspect mode sets value in storage', async ({page, createDraftDocument}) => {
```

Comment:
Is there a need for the same key deletion logic here as we have in the other tests?

Reply:
It's not really necessary since we are checking the put request and asserting on the body of that so whatever was previously there doesn't matter

Reply:
going to merge it but happy to address it in future PR if you think otherwise